python/Scripts/generateShader.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
```

Comment:
This has a new ESSL generation option to test allow usage of Python to generate ESSL shaders.